tests/api/test_millenium_patron.py

```diff
@@ -142,6 +148,24 @@ def test__remote_patron_lookup_success_nonsensical_labels(self):
         assert "alice@sheldon.com" == patrondata.email_address
         assert PatronData.NO_VALUE == patrondata.block_reason
 
+    def test__remote_patron_lookup_success_alternative_identifier(self):
+        self.api = self.mock_api(field_used_as_patron_identifier="pu")
+        self.api.enqueue("dump.success_alternative_identifier.html")
+        patrondata = PatronData(authorization_identifier="good barcode")
+        patrondata = self.api._remote_patron_lookup(patrondata)
+
+        # The barcode is correctly captured from the "NONSENSE[pb]" element.
+        # This checks that we care about the 'pb' code and don't care about
+        # what comes before it.
```

Comment:
It looks like this comment was copied from the previous test and hasn't been updated to reflect the enqueued file or alternative `pu` identifier field for this test.